data/migrations/20211102201230_create_virtualParent_table.js

```diff
@@ -0,0 +1,15 @@
+
+exports.up = function(knex) {
+    return knex.schema.createTable('VirtualParent',function(table){
+        table.increments();
+        table.string('email').notNullable();
+        table.string('first_name').notNullable();
+        table.string('last_name').notNullable();
+    })
+  
+};
+
+exports.down = function(knex) {
+    return knex.schema.dropTable('VirtualParent')
+  
+};
```

Comment:
Please delete this file for the same reason I mentioned in the virtualChild seed file.